x-pack/test/rule_registry/common/config.ts

```diff
@@ -83,6 +84,9 @@ export function createTestConfig(name: string, options: CreateTestConfigOptions)
           ...disabledPlugins
             .filter((k) => k !== 'security')
             .map((key) => `--xpack.${key}.enabled=false`),
+          ...findTestPluginPaths([
+            path.resolve(__dirname, '../../alerting_api_integration/common/plugins'),
```

Comment:
hmmm ... I guess RR must be using the routes as well?  But I'd think they'd be hitting an `_internal` route vs an `api` one.

Reply:
They are using the find event log route too, they are using the event log function in  `x-pack/test/alerting_api_integration/common/lib/get_event_log.ts` that calls the api function now